radio/src/gui/128x64/model_setup.cpp

```diff
@@ -1777,14 +1777,16 @@ void menuModelSetup(event_t event)
       case ITEM_MODEL_SETUP_EXTERNAL_MODULE_OPTIONS:
 #endif
       {
+        #define MULTIMODULE
```

Comment:
er, wth going on there? 😆

Reply:
I put all blame on my helper monkey. Told him to remove stuff for testing the other monkey (me) put in.